### PR TITLE
Buff alpha, beta and gamma tile damage

### PR DESCRIPTION
**Pull request by TranquillyUnpleasant**
So players dont start off being helpless, since you cant manually fight back the flood at all at the start.

### CODE REVIEW

**Comment by fuzzbuck** [comment]
0.5f seems more reasonable

File: core/src/mindustry/content/UnitTypes.java [hunk 1]
```diff
@@ -1340,7 +1340,7 @@ public void load(){
                     lifetime = 60f;
                     shootEffect = Fx.shootSmall;
                     smokeEffect = Fx.shootSmallSmoke;
-                    tileDamageMultiplier = 0.01f;
+                    tileDamageMultiplier = 2f;
```
Comment (TranquillyUnpleasant) [inline core/src/mindustry/content/UnitTypes.java]:
```suggestion
                    tileDamageMultiplier = 0.5f;
```

File: core/src/mindustry/content/UnitTypes.java [hunk 2]
```diff
@@ -1382,7 +1382,7 @@ public void load(){
...
```
Comment (TranquillyUnpleasant) [inline core/src/mindustry/content/UnitTypes.java]:
```suggestion
                    tileDamageMultiplier = 0.5f;
```

File: core/src/mindustry/content/UnitTypes.java [hunk 3]
```diff
@@ -1422,7 +1422,7 @@ public void load(){
                     lifetime = 70f;
                     shootEffect = Fx.shootSmall;
                     smokeEffect = Fx.shootSmallSmoke;
-                    tileDamageMultiplier = 0.01f;
+                    tileDamageMultiplier = 2f;
```
Comment (TranquillyUnpleasant) [inline core/src/mindustry/content/UnitTypes.java]:
```suggestion
                    tileDamageMultiplier = 0.5f;
```

**Comment by TranquillyUnpleasant** [comment]
Done

**Pull request merged**